src/zc/buildout/easy_install.py

```diff
@@ -513,6 +516,9 @@ def _get_dist(self, requirement, ws, for_buildout_run=False):
                     raise zc.buildout.UserError(
                         "Couldn't download distribution %s." % avail)
 
+                if dist.location.endswith('.whl'):
+                    dist = wheel_to_egg(dist, tmp)
```

Comment:
That's probably the best approach. An alternative would be to reduce the precedence of wheels to 2, in which case the egg would be used next time, an advantage being that it requires less of a code change. :)

Note that this is largely moot. Anyone who sets up an egg cache is likely to set up a shared egg directory, so they're not going to be constantly re-processing the wheel.